.github/workflows/deploy.yml

```diff
@@ -1,3 +1,5 @@
+# TODO: name of this file is very confusing, we have in the project repos deploy.yml which calls the shared-build-deploy-container.yml
```

Comment:
This `deploy.yml` is meant to deploy this particular repo.
It's not a .NET solution, so it makes sense that it does different things.

Maybe we should refine the names in each repo:
`deploy-actions.yml`
`deploy-dotnet.yml`

Reply:
Yes, exactly. Unfortunately we can't create nested folders here to organise better 😞